distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/restito/ChunkedContent.java

```diff
@@ -38,6 +40,9 @@
 public class ChunkedContent {
   protected static final Logger LOGGER = LoggerFactory.getLogger(ChunkedContent.class);
 
+  /** Hide the no-arg constructor */
```

Comment:
Just curious, what is this for?

Reply:
To hide the default constructor. When I did it, I forgot the correct term was "default". So I wrote no argument.
It prevents developers from creating an incompletely initialized instance of the class.